docs/config_search.md

```diff
@@ -117,12 +117,18 @@ You can also modify the minimum/maximum values that the automatic search space w
 
 ---
 
-### [Request Concurrency Search Space](https://github.com/triton-inference-server/client/blob/main/src/c++/perf_analyzer/README.md#request-concurrency)
+### [Request Concurrency Search Space](https://github.com/triton-inference-server/client/blob/main/src/c++/perf_analyzer/docs/inference_load_modes.md#concurrency-mode))
```

Comment:
Was this link broken? Shouldn't our CI be catching this?

Reply:
Link wasn't broken. It just wasn't as accurate as it could have been. Now we go directly to the page with the info about concurrency.